source/dotnet/Library/CppConsoleApp/packages.config

```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="AdaptiveCards.Template" version="1.0.0" targetFramework="native" />
```

Comment:
is this line presuming the existence of the AdaptiveCards.Template package that supports C++ code?

Reply:
nuget package is generated by the new template project.